modules/nf-core/hlala/preparegraph/main.nf

```diff
@@ -0,0 +1,33 @@
+process HLALA_PREPAREGRAPH {
+    label 'process_high'
+
+    conda (params.enable_conda ? "bioconda::hla-la=1.0.3" : null)
```

Comment:
```suggestion
    conda "bioconda::hla-la=1.0.3"
```

Reply:
wasn't there the problem that it does not work with conda and we wanted to add a warning here?